bigquery_etl/query_scheduling/templates/airflow_dag.j2

```diff
@@ -85,6 +91,12 @@ with DAG('{{ name }}', default_args=default_args{%+ if schedule_interval != None
             {%+ if task.email_on_retry != None -%}
             email_on_retry={{ task.email_on_retry }},
             {%+ endif -%}
+            {%+ if task.sla != None -%}
```

Comment:
```suggestion
            {%+ if task.sla is not none -%}
```

Jinja has a `none` test: https://jinja.palletsprojects.com/en/2.11.x/templates/#none

My intuition says we should be using that rather than `!= None`, but I don't fully understand the implications in the Jinja context.